answerking_app/models/serializers.py

```diff
@@ -25,7 +24,71 @@ def compress_white_spaces(value: str) -> str:
     return re.sub(" +", " ", value)
 
 
-class ItemSerializer(serializers.ModelSerializer):
+class CategoryDetailSerializer(serializers.ModelSerializer):
+    id = serializers.IntegerField(required=False)
+    name = serializers.CharField(
+        max_length=50,
+        allow_blank=False,
+        validators=[RegexValidator("^[a-zA-Z !]+$")],
+    )
+    description = serializers.CharField(
+        max_length=200,
+        allow_null=True,
+        allow_blank=True,
+        validators=[
+            RegexValidator(
+                "^[a-zA-Z .!,#]+$",
```

Comment:
why are we allowing # in the description?

Reply:
'The world's #1 Cheeseburger' ?

Reply:
I dont mind we can remove it whatever people think

Reply:
Hashtags on social media?